.gitignore

```diff
@@ -19,3 +19,6 @@ dependencies/*
 
 # Ignore log files from OpenSim crashing
 *.log
+
+# Ignore OSX attribute files
```

Comment:
The appearance of these files in git might be due to mixing operating systems, like accessing the repo, stored on an OS X drive, via Windows in Parallels?

You can maybe add .DS_Store to the .gitignore file.